lib/spack/docs/analyze.rst

```diff
@@ -9,30 +9,43 @@
 Analyze
 =======
 
-The analyze commands allows you to take an existing package install, and run
-an analyzer to extract one or more metrics. The metrics can range from reading
-metadata about install files and output to extracting Application Binary
-Interface (ABI) information from binaries. 
+
+The analyze command is a front-end to various tools that let us analyze
+package installations. Each analyzer is a module for a different kind
+of analysis that can be done on a package installation, including (but not
+limited to) binary, log, or text analysis. Thus, the analyze command group
+allows you to take an existing package install, choose an analyzer,
+and extract some output for the package using it.
+
 
 -----------------
 Analyzer Metadata
 -----------------
 
-For all analyzers, we write to an ``analyze`` folder in the package install
-directory metadata folder. For example, here we see an analyze folder that is
-alongside a wget install:
+For all analyzers, we write to an ``analyzers`` folder in your spack ``$HOME``
+directory, or the value that you specify in your spack config at 
+``config:analyzers_dir``. We don't want to write to your package install folder
+in the case that it is read only. For example, here we see the results
+of running an analysis on zlib:
 
 .. code-block:: console
 
-    $ ls .spack/
-    analyze                   archived-files                     repos                              spack-build-03-build-out.txt
-    spack-build-out.txt       install_environment.json           spack-build-01-autoreconf-out.txt  spack-build-04-install-out.txt
-    spack-configure-args.txt  install_manifest.json              spack-build-02-configure-out.txt   spack-build-env.txt
-    spec.yaml
+    $ tree ~/.spack/analyzers/
+    /home/vanessa/.spack/analyzers/
```

Comment:
I guess we know you wrote this :).  In other parts of the docs -- it's a little inconsistent -- we've used `myuser`, `spackuser`, `username`, and `user`... if you want to anonymize feel free.  @alalazo is in the modules reference as `mculpo`.  Wonder if we should standardize that.

Reply:
yes we probably should, if @alalazo is okay with it. I would opt for the most fun of the options (so not user or username, but maybe spackuser or similar).

Reply:
I updated here to be spackuser, and we can discuss if there should be a global strategy!